app/assets/stylesheets/components/layout.scss

```diff
@@ -189,3 +189,14 @@ i.fa {
     border-right: 1px solid $primary-colour;
   }
 }
+
+//popover css used in signup page for client side validation
+
+#signup_page .popover{
+    background-color: rgba(0,0,0,0.6);

```

Comment:
@vitoravelino I don't think it's ok to override it. All popovers should look and feel in the same way throughout the app.